scripts/Level.gd

```diff
@@ -0,0 +1,9 @@
+extends Node
+
+export(String) var level_name
+
+signal start_level
```

Comment:
Is this used? If not I would probably not add it (yet).

Reply:
No, it's just leftover of me trying out signals and then realizing we won't use them in this case. Will remove it.